FAIL: test_simple_from_running_python  on OSX

Running tests on OSX raises the following error

```python
======================================================================
FAIL: test_simple_from_running_python (okonomiyaki.runtimes.tests.test_runtime.TestPythonRuntime)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/.../.edm/envs/testing/lib/python2.7/site-packages/okonomiyaki/runtimes/tests/test_runtime.py", line 39, in test_simple_from_running_python
    os.path.realpath(NORM_EXECUTABLE)
AssertionError: u'/Users/mdickinson/.edm/envs/testing/bin/python2.7' != u'/Users/.../.edm/envs/testing/Resources/Python.app/Contents/MacOS/Python [truncated]...
- /Users/.../.edm/envs/testing/bin/python2.7
+ /Users/.../.edm/envs/testing/Resources/Python.app/Contents/MacOS/Python
```

--- a/okonomiyaki/runtimes/tests/test_runtime.py
+++ b/okonomiyaki/runtimes/tests/test_runtime.py
@@ -26,18 +26,19 @@ class TestPythonRuntime(unittest.TestCase):
 
         # Then
         self.assertEqual(runtime_info.prefix, NORM_EXEC_PREFIX)
-        if sys.platform == "win32":
+        if sys.platform.startswith("win"):
             # XXX: we can't easily check whether two paths are the same file on
             # windows on python 2, as os.path.samefile is not available there
             self.assertEqual(
                 os.path.realpath(runtime_info.executable).lower(),
                 os.path.realpath(NORM_EXECUTABLE).lower()
             )
+        elif sys.platform.startswith("darwin") and sys.version_info[:2] == (2, 7):
+            raise unittest.SkipTest('sys.executable returns incompatible path, see #294')
         else:
             self.assertEqual(
                 os.path.realpath(runtime_info.executable),
-                os.path.realpath(NORM_EXECUTABLE)
-            )
+                os.path.realpath(NORM_EXECUTABLE))
 
     @unittest.skipIf(
         hasattr(sys, "pypy_version_info"),

